learn/using-the-cli-tools.md

```diff
@@ -123,15 +119,39 @@ These powerful supporting tools extend Ballerina to various ecosystem technologi
 </td>
 </tr>
 <tr>
+<td class="cCommand">grpc</td>
+<td class="cDescription">This is the gRPC stub/skeleton generation tool. For more information, see <a href="/learn/generating-ballerina-code-for-protocol-buffer-definitions">Generating Ballerina Code for Protocol Buffer Definitions</a>.</td>
+</tr>
+<tr>
 <td class="cCommand">openapi</td>
-<td class="cDescription">This is the OpenAPI (Swagger) stub/skeleton generation tool. For more information, see <a href="/learn/how-to-use-openapi-tools">Ballerina OpenAPI Tools</a>.</td>
+<td class="cDescription">This is the OpenAPI (Swagger) stub/skeleton generation tool. For more information, see <a href="/learn/using-the-openapi-tools">Using the OpenAPI Tools</a>.</td>
 </tr>
 <tr>
-<td class="cCommand">grpc</td>
-<td class="cDescription">This is the gRPC stub/skeleton generation tool. For more information, see <a href="/learn/how-to-generate-code-for-protocol-buffers">How to generate Ballerina code for Protocol Buffer Definition</a>.</td>
+<td class="cCommand">version</td>
+<td class="cDescription">Tells you the version of Ballerina you have installed.</td>
```

Comment:
@praneesha ; Should we explain this further? That versions shows currently using distribution version and based language specific version. Also it include update tool version as well

Reply:
Done.